lib/fast_jsonapi/attribute.rb

```diff
@@ -20,7 +20,7 @@ def serialize(record, serialization_params, output_hash)
 
     def include_attribute?(record, serialization_params)
       if conditional_proc.present?
-        conditional_proc.call(record, serialization_params)
+        conditional_proc.arity.abs == 1 ? conditional_proc.call(record) : conditional_proc.call(record, serialization_params)
```

Comment:
Since this check would run when serializing every object it could be expensive. Instead could we explore just setting a default `serialization_params = {}`. That might be faster and might solve the problem.